presto-hive/src/main/java/com/facebook/presto/hive/HiveDirectoryContext.java

```diff
@@ -13,17 +13,28 @@
  */
 package com.facebook.presto.hive;
 
+import com.google.common.collect.ImmutableMap;
+
+import java.util.Map;
+
 import static java.util.Objects.requireNonNull;
 
 public class HiveDirectoryContext
 {
+    public static final String PRESTO_QUERY_ID = "presto_query_id";
+    public static final String PRESTO_QUERY_SOURCE = "presto_query_source";
+    public static final String PRESTO_CLIENT_INFO = "presto_client_info";
+    public static final String PRESTO_USER_NAME = "presto_user_name";
+    public static final String PRESTO_METASTORE_HEADER = "presto_metastore_header";
```

Comment:
Wondering if it makes sense to move these to HiveUtils ...so that `buildDirectoryContextProperties()` is self sufficient inside HiveUtil